source/common/res/features/.eslintignore

```diff
@@ -0,0 +1,4 @@
+**/*/reports/chartjs-2.0.0-beta2/
+**/*/resize-inspector/jquery-resizable.min.js
+**/*/reports/noUiSlider.8.2.1/nouislider.min.js
+allSettings.js
```

Comment:
Looks like I should make the output from the Python script comply with our linting rules!